service/src/main/resources/application.yaml

```diff
@@ -32,7 +32,7 @@ fees-register:
       event: general application
       jurisdiction1: family
       jurisdiction2: family court
-      keyword: without-notice
+      keyword: ${FEES_REGISTER_C2_WITHOUT_NOTICE_KEYWORD:GeneralAppWithoutNotice}
```

Comment:
Don't think that the whole `${FEES_REGISTER_C2_WITHOUT_NOTICE_KEYWORD:GeneralAppWithoutNotice}` is needed, just `GeneralAppWithoutNotice`.
From my understanding as the env variable should map directly to the property spring should recognise this and give the env variable priority over the `application.yaml` value.

Reply:
Ahh it would need to be called `FEES_REGISTER_PARAMETERS_C2_WITHOUT_NOTICE_KEYWORD`

Reply:
@lewisbirks are you sure about this? Code does not directly goes this this property

Reply:
Chatted with tomasz and we can leave as is, just more work for a temp thing than is needed.

Reply:
i would prefer to keep explicit env variable, if somebody decide to refactor name in application xml then  env variable wont match and prod config can be broken without even noticing it. 
Thats nice feature, but i guess quite risky if you dont now context

Reply:
@nkoranne  As a safer approach we could leave old default and ovewrite with the new value in flux demo. 

But that would work if the overwriting is confirmed and we have got flux ready for prod to use the old value. (which can be merged beforehand)